### PR TITLE
Add Specific Shop for Shop Owner feature

### PR DESCRIPTION
**Pull request by stringtheory1234**

### CODE REVIEW

File: frontendservice/src/views/ShopOwnerSpecificShop.vue [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  props: ["id"],
+  data() {
+    return {
+      shopList: "",
+      tokenList: [],
+      renderComponent: true
+    };
+  },
+  methods:{
+    deleteToken: function(tokenId) {
```
Comment (sdumka) [inline frontendservice/src/views/ShopOwnerSpecificShop.vue:67]: Can you add a deletion confirmation (similar to delete shops) here?

Comment (stringtheory1234) [reply frontendservice/src/views/ShopOwnerSpecificShop.vue:69]: yes added!

File: frontendservice/src/views/ShopOwnerSpecificShop.vue [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        title: "Are you sure?",
+        text: "You will not be able to revert this!",
+        icon: "warning",
+        showCancelButton: true,
+        confirmButtonColor: "#3085d6",
+        cancelButtonColor: "#d33",
+        confirmButtonText: "Yes, delete it!"
+      }).then(result => {
+        if (result.isConfirmed == true) {
+          console.log("hi there!");
```
Comment (sdumka) [inline frontendservice/src/views/ShopOwnerSpecificShop.vue]: remove or log something more informative?

Comment (stringtheory1234) [reply frontendservice/src/views/ShopOwnerSpecificShop.vue]: Sorry for that had to remove before pushing!

**Pull request merged**